priv/perf/apps/load_test/lib/scenario/account_transactions.ex

```diff
@@ -0,0 +1,271 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Scenario.AccountTransactions do
+  @moduledoc """
+  This scenario tests watcher info apis when number of transaction increases for an account.
+  """
+
+  use Chaperon.Scenario
+
+  alias Chaperon.Timing
+  alias LoadTest.Connection.WatcherInfo, as: Connection
+  alias LoadTest.Service.Faucet
+  alias LoadTest.Ethereum.Account
+  alias LoadTest.Utils.Encoding
+  alias WatcherInfoAPI.Api
+  alias WatcherInfoAPI.Model
+
+  @poll_interval 15_000
+  @default_retry_attempts 15
+
+  @eth <<0::160>>
+  @test_output_amount 1
+
+  @spec init(Chaperon.Session.t()) :: Chaperon.Session.t()
+  def init(session) do
+    session
+    |> log_info("start init with random delay...")
+    |> random_delay(Timing.seconds(5))
+  end
+
+  def run(session) do
+    iterations = config(session, [:iterations])
+    fee_wei = Application.fetch_env!(:load_test, :fee_wei)
+
+    amount = iterations * (@test_output_amount + fee_wei)
+    {:ok, sender} = Account.new()
+    {:ok, _} = Faucet.fund_child_chain_account(sender, amount, @eth)
+
+    {:ok, faucet} = Faucet.get_faucet()
+
+    session
+    |> assign(faucet: faucet, iteration: 1)
```

Comment:
As I understand faucet is just a recipient of funds used in `create_transaction/2` how about we call this session value `recipient`?

Reply:
It is, but I kind of like that calling it faucet makes it obvious that the funds are being returned to the faucet.
I don't feel strongly about it though 🤷‍♂